app/controllers/application_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ApplicationController < ActionController::Base
+  include ApplicationHelper
+  def signout
```

Comment:
This method is missing something very important. It needs to be able to update the `remember_token`'s value in the database to `nil`. This could be achieved with a method in the User model that you can call from here with `current_user.update_attributes(:remember_digest, nil)`.

[**REQUIRED**]